modules/swagger-core/src/test/java/io/swagger/BeanValidatorTestNG.java

```diff
@@ -0,0 +1,37 @@
+package io.swagger;
+
+import io.swagger.converter.ModelConverters;
+import io.swagger.models.Model;
+import io.swagger.models.properties.DoubleProperty;
+import io.swagger.models.properties.IntegerProperty;
+import io.swagger.models.properties.Property;
+import io.swagger.models.properties.StringProperty;
+import models.BeanValidationsModel;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+public class BeanValidatorTestNG extends Assert {
```

Comment:
I don't like the idea with renaming and TestNG ending. There is nothing wrong if some tests will run twice (of course until they will not fail). This just means that these tests are first candidates for migration.
But my main concern here that we introduce implicit naming convention for Java tests for other community members.